src/utilities/show.jl

```diff
@@ -176,7 +176,13 @@ end
 
 function show(io::IO, p::Problem)
     TreePrint.print_tree(io, p, MAXDEPTH[])
-    print(io, "\ncurrent status: $(p.status)")
+    if p.status == MOI.OPTIMIZE_NOT_CALLED
+        print(io, "\nstatus: `solve!` not called yet")
+    else
+        print(io, "\ntermination status: $(p.status)")
+        print(io, "\nprimal status: $(primal_status(p))")
+        print(io, "\ndual status: $(dual_status(p))")
```

Comment:
MOI returns us lots of status information (termination status, primal status, dual status), and we might as well give it all to the user, otherwise they probably won't know it's there.